modules/services/src/main/scala/cards/nine/services/free/interpreter/collection/Services.scala

```diff
@@ -23,22 +23,22 @@ import cards.nine.domain.application.Package
 import cards.nine.domain.pagination.Page
 import cards.nine.services.common.PersistenceService
 import cards.nine.services.common.PersistenceService._
-import cards.nine.services.free.algebra.SharedCollection._
+import cards.nine.services.free.algebra.Collection._
 import cards.nine.services.free.domain.SharedCollection.Queries
 import cards.nine.services.free.domain._
 import cards.nine.services.free.interpreter.collection.Services.SharedCollectionData
 import cards.nine.services.persistence.Persistence
 import cats.syntax.either._
-import cats.{ Monad, ~> }
+import cats.Monad
 import doobie.contrib.postgresql.pgtypes._
 import doobie.imports.ConnectionIO
 import shapeless.syntax.std.product._
 
 class Services(
   collectionPersistence: Persistence[SharedCollection]
-)(implicit connectionIOMonad: Monad[ConnectionIO]) extends (Ops ~> ConnectionIO) {
+)(implicit connectionIOMonad: Monad[ConnectionIO]) extends Handler[ConnectionIO] {
```

Comment:
I like it better when we `extends Collection.Handler[...]` because `Handler` is a path dependent type and if there were multiple Handlers defined here you'd have to namespace them anyway.